api/ShadowRoot.json

```diff
@@ -704,9 +704,13 @@
       "setHTMLUnsafe": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/ShadowRoot/setHTMLUnsafe",
-          "spec_url": "https://html.spec.whatwg.org/multipage/dynamic-markup-insertion.html#dom-shadowroot-sethtmlunsafe",
+          "spec_url": [
```

Comment:
Whatever is agreed in https://github.com/mdn/browser-compat-data/pull/26659/files#r2071197146 should be done here too - ie. this is implemented but without santiser option.

NOTE also, there is a ShadowRoot.setHTML, but I don't THINK that is implemented anywhere